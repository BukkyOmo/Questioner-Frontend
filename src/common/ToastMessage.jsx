import React from 'react';

const ToastMessage = ({ message }) => (
  <p className='toast-message'>{message}</p>
);

export default ToastMessage;
