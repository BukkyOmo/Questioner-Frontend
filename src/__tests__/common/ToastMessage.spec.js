import React from 'react';
import { shallow } from 'enzyme';
import ToastMessage from '../../common/ToastMessage';

describe('<ToastMessage />', () => {
  it('renders p element', () => {
    const wrapper = shallow(<ToastMessage />)
    expect(wrapper.find('p').exists()).toBe(true);
  });
   it('renders correctly using snapshot', () => {
    expect(ToastMessage).toMatchSnapshot();
   });
});
