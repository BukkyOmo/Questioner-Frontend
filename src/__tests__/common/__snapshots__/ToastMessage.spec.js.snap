// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToastMessage /> renders correctly using snapshot 1`] = `[Function]`;
